pkgs/servers/aws-codeartifact-proxy/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "aws-codeartifact-proxy";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "sktan";
+    repo = "aws-codeartifact-proxy";
+    rev = "v${version}";
+    hash = "sha256-mHXqwH6C6jUighkeRUu4H30z0o+Y0Zh7aE6foGZ2gKY=";
+  } + "/src";
+
+  vendorSha256 = "3MO+mRCstXw0FfySiyMSs1vaao7kUYIyJB2gAp1IE48=";
```

Comment:
```suggestion
  vendorHash = "sha256-3MO+mRCstXw0FfySiyMSs1vaao7kUYIyJB2gAp1IE48=";
```

Reply:
Implemented in b4ea51d6d2fb28ea7bb20dbbe5705d853ad0c6a2.